Jenkinsfile

```diff
@@ -279,11 +280,26 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                                 checkout scm
                                             }
 
-                                        println "Test OS X"
-                                            // Retry to protect against flakes
-                                            retry(3) {
-                                                testNixGo("OS X")
+                                        parallel(
+                                            test_RN: {
+                                                println "Test React Native"
+                                                dir("react-native") {
+                                                    sh "npm i"
+                                                    sh "npm run gobuild-ios"
+                                                    lock('iossimulator') {
```

Comment:
should be `"iossimulator${NODE_NAME}"`, otherwise it'll be a global lock over all of Jenkins.